bundle.prod/manifests/logic-operator-rhel8-builder-config_v1_configmap.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+data:
+  DEFAULT_BUILDER_RESOURCE_NAME: Dockerfile
```

Comment:
I think I've removed this line, but we can fix this if after this PR gets merged. Ideally, the `logic-*` files should be copied from the upstream part and apply a template overriding prod unique info.